react/ConditionLayoutCategory.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useMemo } from 'react'
+import type { ComponentType } from 'react'
+import { useRuntime } from 'vtex.render-runtime'
+
+import ConditionLayout from './ConditionLayout'
+import type { NoUndefinedField, MatchType, Condition, Handlers } from './types'
+
+type Props = {
+  conditions: Array<Condition<ContextValues, HandlerArguments>>
+  matchType?: MatchType
+  Else?: ComponentType
+  Then?: ComponentType
+}
+
+type ContextValues = {
+  id: string,
+  type: 'category' | 'department'
+}
+
+type HandlerArguments = {
+  categoryId: { ids: string[] }
+  departmentId: { ids: string[] }
+}
+
+const HANDLERS: Handlers<ContextValues, HandlerArguments> = {
```

Comment:
What do you think about this suggestion?
```suggestion
const handlersMap: Handlers<ContextValues, HandlerArguments> = {
```